mappings/net/minecraft/nbt/StringNbt.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_2519 net/minecraft/nbt/StringTag
+CLASS net/minecraft/class_2519 net/minecraft/nbt/StringNbt
```

Comment:
Isn't this a class rename of a core NBT type ?

Reply:
yup needs fixing. guess I missed reverting that one